### DISCUSSION TITLE
Themes…

### DISCUSSION THREAD

**Discussion by Hermann12** [KDE/ghostwriter#841] [General]
I have updated to V2.1.6 on Win10 Portable.
Now I have a strange behaviour, if I open Themes from the left below wheel, it opens a window which is very huge. I can't close it, because it seem it is 3 times width as my used screen and sometimes the program freeze, I have a resolution 3840x2060, maybe this cause this behaviour, or is there a setting what I miss?

**Comment by Sci-Mon** [comment]
Same here on my 4k monitor/resolution... I am using XFCE/XFWM, so I was able finally to close the dialog by pressing ALT and dragging the dialogfurther up until you can see the CLOSE or EDIT button. (Just a workaround...)
